src/test/scala/uk/gov/hmrc/test/ui/pages/Arrival/TransportEquipmentAndSeal/AddAnotherContainerSealPage.scala

```diff
@@ -21,7 +21,9 @@ import uk.gov.hmrc.test.ui.pages.YesNoPage
 object AddAnotherContainerSealPage extends YesNoPage {
 
   override def title(args: String*): String = args match {
-    case Seq(numberOfSeals, identificationVal) if(numberOfSeals == "1") => String.format("You have added %s seal for container %s", numberOfSeals, identificationVal)
-    case Seq(numberOfSeals, identificationVal) => String.format("You have added %s seals for container %s", numberOfSeals, identificationVal)
+    case Seq(numberOfSeals, identificationVal) if numberOfSeals == "1" =>
```

Comment:
I know this is just a fmt change but this can be refactored to `case Seq("1", identificationVal)` if you can be bothered 😄

Reply:
Agreed to leave as is for now as there's an upcoming change in a different branch